app/views/submissions/index.html.erb

```diff
@@ -19,43 +19,62 @@
 <hr>
 
 <div class="row buttons-row">
-    <div class="buttons-spacing">
-    <%= link_to "<i class='material-icons left submissions-icons'>add</i>Create Submission".html_safe,
-        new_course_assessment_submission_path(@course, @assessment),
-        {:title=>"Create a new submission for a student, with an option to submit a handin file on their behalf",
-         :class=>"btn submissions-main"} %>
-    </div>
-
-    <div class="buttons-spacing">
+  <div class="buttons-spacing">
+    <%= link_to "<i class='material-icons left submissions-icons'>add</i>Create Submission".html_safe.html_safe,
+                new_course_assessment_submission_path(@course, @assessment),
+                { title: "Create a new submission for a student, with an option to submit a handin file on their behalf",
+                  class: "btn submissions-main" } %>
+  </div>
+  <div class="buttons-spacing">
     <%= link_to "<i class='material-icons left submissions-icons'>file_download</i>Download Final Submissions".html_safe,
-        downloadAll_course_assessment_submissions_path(@course, @assessment, final: true),
-        {:title=>"Download final submissions from each student",
-         :class=>"btn submissions-main"} %>
-    </div>
-
-    <div class="buttons-spacing"> <!-- TODO: Change functionality to filter for missing submissions -->
+                downloadAll_course_assessment_submissions_path(@course, @assessment),
+                { title: "Down all submissions from each student",
+                  class: "btn submissions-main" } %>
+  </div>
+  <div class="buttons-spacing">
     <%= link_to "<i class='material-icons left submissions-icons'>people</i>Missing Submissions".html_safe,
-        missing_course_assessment_submissions_path(@course, @assessment),
-        {:title=>"List the students who have not submitted anything",
-         :class=>"btn submissions-main"} %>
-    </div>
-
-    <div class="buttons-spacing">
+                downloadAll_course_assessment_submissions_path(@course, @assessment, final: true),
+                { title: "Download the most recent submission from each student",
+                  class: "btn submissions-main" } %>
+  </div>
+  <div class="buttons-spacing">
     <%= link_to "<i class='material-icons left submissions-icons'>event</i>Manage Extensions".html_safe,
-        [@course, @assessment, :extensions],
-        {:class=>"btn submissions-main"} %>
+                missing_course_assessment_submissions_path(@course, @assessment),
```

Comment:
These three buttons are linked to the wrong path
- download final -> download all
- download all -> (missing)
- missing submissions -> download final
- manage extensions -> missing submissions
- (missing) -> manage extensions